nuklear.h

```diff
@@ -12895,6 +12895,7 @@ nk_font_bake(struct nk_font_baker *baker, void *image_memory, int width, int hei
                 dst_font->ascent = ((float)unscaled_ascent * font_scale);
                 dst_font->descent = ((float)unscaled_descent * font_scale);
                 dst_font->glyph_offset = glyph_n;
+                dst_font->glyph_count = 0;
```

Comment:
Could you please add comment, why the zeroing is necessary? It's currently not obvious without reading the description in your pull request :wink:.

Reply:
Okie dokes. Added the requested changes. Please review.